src/core/credentials/transport/tls/spiffe_utils.h

```diff
@@ -28,6 +28,8 @@
 
 namespace grpc_core {
 
+std::string BundleRootToPem(absl::string_view bundle_root);
```

Comment:
The variable name "bundle_root" is ambiguous - can you clarify what this will actually be? Is it just a DER-encoded cert? If yes, can we clarify the function name and variable name?

Reply:
Thanks SGTM.